modules/puppet_connect/README.md

```diff
@@ -0,0 +1,26 @@
+# puppet_connect
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Requirements](#requirements)
+3. [Usage](#usage)
+4. [Reference](#reference)
+
+## Description
+
+This module provides the puppet_connect::test_input_data plan, which is used to test that the provided Puppet Connect input data is complete. The module should be used exactly as specified in the Puppet Connect docs since it is designed for a highly specialized workflow.
```

Comment:
I don't think the README needs to be more detailed than this since all its possible viewers (user and maintainer) will likely be viewing the Puppet Connect docs when interacting with this module.